pkg/controller/registry/resolver/querier.go

```diff
@@ -24,20 +28,20 @@ type SourceRef struct {
 }
 
 type SourceQuerier interface {
-	FindProvider(api opregistry.APIKey, initialSource CatalogKey) (*api.Bundle, *CatalogKey, error)
+	FindProvider(api opregistry.APIKey, initialSource CatalogKey, pkgName string) (*api.Bundle, *CatalogKey, error)
```

Comment:
Again, `pkgName` feels deceptive. What do we think about something like `dependentPkgName`?

Reply:
I will go with excludingPkgName.